src/routes/[link=link].svg/+server.ts

```diff
@@ -0,0 +1,23 @@
+import type { RequestHandler } from './$types';
+import { LINKS } from '$lib/links/data';
+import { parseLink, genQRCodeSvg } from '$lib/links/utils';
+
+export const GET = handler();
+
+function handler(): RequestHandler {
+  return async function (event) {
+    const link = parseLink(event.params.link);
```

Comment:
Nit: Naming is confusing.
```suggestion
    const id = parseLinkId(event.params.link);
```

Reply:
Done.